includes/GameController.hpp

```diff
@@ -126,6 +131,11 @@ class GameController : public Controller
        false */
     bool playingPeriod {false};
 
+    /* this boolean is true when the player just loses the current game; it is
+       used to detect when the player has no life anymore and when the lose
+       screen has to be displayed for a short time */
+    bool losePeriod {false};
```

Comment:
the losePeriod boolean and the startLosePeriodTime can be refactored... if the startLosePeriodTime is strictly positive, then that means the game is currently inside the lose period